scripts/app.js

```diff
@@ -2,4 +2,6 @@
 
 function Project(name) {
   this.name = name;
+  this.group = group;
```

Comment:
These are causing issues. Why? (Hint: see the properties you put into your constructor)